### DISCUSSION TITLE
What GPU are you all using

### DISCUSSION THREAD

**Discussion by xiasi0** [SYSTRAN/faster-whisper#775] [Q&A]
Please recommend an affordable GPU

**Comment by etlweather** [comment]
We don't know what affordable means for you, neither what your intended use case and performance requirements are. If it is for a personal project, is $2,000 affordable? If it is for a major business venture, would $1M be affordable? Is 1:1 audio length to transcription time acceptable? Is 4:1, 10:1 acceptable? Does your audio have a lot of silence gap in it or is constant talk? etc.
